### ISSUE TITLE
ICNet

### ISSUE THREAD

**Issue by gitunit** [BBuf/Keras-Semantic-Segmentation#22]
isn't this implementation of ICNet missing the **Cascade Label Guidance**? (section 3.4 in the paper)

also i can't see the **weighted softmax cross entropy loss** which should be applied to each branch (see definition 2 in the paper)

pls correct me if I'm wrong.
